modules/pipelines-document-attributes.adoc

```diff
@@ -9,4 +9,4 @@
 //
 :pipelines-title: Red Hat OpenShift Pipelines
 :pipelines-shortname: Pipelines
-:pipelines-ver: pipelines-1.3
+:pipelines-ver: pipelines-1.4
```

Comment:
Please note @sounix000 is updating this via his PR. One of you is likely to get a merge conflict. I would suggest we get this in, as this is smaller than his PR.

Reply:
ok, thanks.